src/Console/Relay.php

```diff
@@ -0,0 +1,134 @@
+<?php
+/**
+ * @copyright Copyright (C) 2020, Friendica
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace Friendica\Console;
+
+use Asika\SimpleConsole\CommandArgsException;
+use Friendica\Model\APContact;
+use Friendica\Model\Contact;
+use Friendica\Protocol\ActivityPub\Transmitter;
+
+/**
+ * tool to control the list of ActivityPub relay servers from the CLI
+ *
+ * With this script you can access the relay servers of your node from
+ * the CLI.
+ */
+class Relay extends \Asika\SimpleConsole\Console
+{
+	protected $helpOptions = ['h', 'help', '?'];
+
+	/**
+	 * @var $dba Friendica\Database\Database
+	 */
+	private $dba;
+
+
+	protected function getHelp()
+	{
+		$help = <<<HELP
+console relay - Manage ActivityPub relay configuration
+Synopsis
+	bin/console relay [-h|--help|-?] [-v]
+	bin/console relay add <actor> [-h|--help|-?] [-v]
+	bin/console relay remove <actoor> [-h|--help|-?] [-v]
+
+Description
+	bin/console relay
+		Lists all active relay servers
+
+	bin/console relay add <actor>
+		Add a relay actor in the format https://relayserver.tld/actor
```

Comment:
If the format is fixed, why not ask just for the server base URL instead?

Reply:
I think this is more flexible. Possibly - for example - we could extend this in the future in a way that we directly connect Friendica servers this way. Then the actor would be different.